README.md

```diff
@@ -85,10 +90,10 @@ The values above 1 for the split Gelman Rubin diagnostic (`r_hat`) indicates tha
 ...                                                             dist.transforms.AffineTransform(mu, tau)))
 ...         numpyro.sample('obs', dist.Normal(theta, sigma), obs=y)
 
->>> nuts_kernel = NUTS(eight_schools)
+>>> nuts_kernel = NUTS(eight_schools_noncentered)
 >>> mcmc = MCMC(nuts_kernel, num_warmup=500, num_samples=1000)
 >>> rng_key = random.PRNGKey(0)
->>> mcmc.run(rng_key, J, sigma, y=y, extra_fields=('diverging',))
+>>> mcmc.run(rng_key, J, sigma, y=y, extra_fields=('potential_energy',))
```

Comment:
In my PR, I set the arg `extra_fields=('diverging',)` by default. So could you make a corresponding change in MCMC.run method, which does
```
extra_fields = set(['diverging']).union(set(extra_fields))
```
to match the behavior here?

Reply:
Oh..I see, I'll do that. 

> Could you also update versions in examples/tutorials too?

Forgot about that; will do.